spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -758,6 +758,16 @@
                 job_name: 'GetUspsProofingResultsJob',
               )
             end
+
```

Comment:
I wonder if it is worth adding a test for when the profile has a deactivation reason present. This test might go away if/when we add code around ignoring enrollments with profiles that are deactivated with reason encryption error and/or password_reset, but a test would at least point out that we are currently processing under those conditions.

Reply:
I had that same thought. I dug around as to where this would go. I realized we don't have specs around pw reset (testing profile, enrollment). That caused me to want to collect current behavior so we could write up a whole suite of specs.

Reply:
@shanechesnutt-ft  See commit 39d49646c23ca1638eaba83808d7c13ee2afdcdc 

Requesting your review and approval again after updates